apps/nextjs-pages-router/src/middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { makeswiftMiddleware } from '@makeswift/runtime/next/middleware'
+
+export default makeswiftMiddleware({
+  apiKey: process.env.MAKESWIFT_SITE_API_KEY!,
```

Comment:
Probably unrelated to this PR, but I don't love that we're using a non-null assertion for environment variables